qa/workunits/rbd/rbd_mirror_journal.sh

```diff
@@ -249,6 +266,12 @@ clone_image ${CLUSTER2} ${PARENT_POOL} ${parent_image} ${parent_snap} ${POOL} \
 test $(get_clone_format ${CLUSTER2} ${POOL} ${clone_image}_v1) = 1
 wait_for_image_replay_started ${CLUSTER1} ${POOL} ${clone_image}_v1
 test $(get_clone_format ${CLUSTER1} ${POOL} ${clone_image}_v1) = 1
+remove_image_retry ${CLUSTER2} ${POOL} ${clone_image}_v1
+remove_image_retry ${CLUSTER1} ${POOL} ${clone_image}1
+wait_for_image_present ${CLUSTER1} ${POOL} ${clone_image}_v1 'deleted'
+wait_for_image_present ${CLUSTER2} ${POOL} ${clone_image}1 'deleted'
+unprotect_snapshot ${CLUSTER2} ${PARENT_POOL} ${parent_image} ${parent_snap}
```

Comment:
I still have some errors on this line sometimes. With a ebusy error, the wait_for_image_present ... 'deleted' are probably not sufficient... Do you have any idea how to prevent this? I had a sleep 10 before to workaround this...

Reply:
A straightforward way would be to add and use `unprotect_snapshot_retry` here.

May be there is a clever way but I can't tell right now, I would need to look why exactly it is still returns EBUSY while `wait_for_image_present` reports it is already deleted. Anyway `unprotect_snapshot_retry` would be good enough as a start.

Reply:
Indeed, thanks it works!